ui/src/app/images/views/ImageList/ImageList.tsx

```diff
@@ -0,0 +1,14 @@
+import Section from "app/base/components/Section";
+import { useWindowTitle } from "app/base/hooks";
+
+const ImagesList = (): JSX.Element => {
+  useWindowTitle("Images");
+
+  return (
+    <Section header="Images" headerClassName="u-no-padding--bottom">
```

Comment:
Removing the padding makes the header look weird, but I imagine this'll get fixed once `SectionHeader` is used instead of plain text

Reply:
Yeah I think so, I just left it like that because that's how the others were.